snapm/manager/_mounts.py

```
@@ -0,0 +1,651 @@
+    a label reference it is returned unmodified.
+
+    :param device: The device string to evaluate; '/dev/...', 'LABEL=...',
+                   or 'UUID=...'.
+    :returns: The device path.
+    :rtype: ``str``
+    """
+    if device.startswith("UUID="):
+        uuid = device.split("=", maxsplit=1)[1]
+        resolved = get_device_path("uuid", uuid)
+        if resolved is None:
+            raise SnapmNotFoundError(f"Device with UUID '{uuid}' not found")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)